modules/ps_contactinfo/ps_contactinfo-rich.tpl

```diff
@@ -53,7 +53,7 @@
     <hr/>
     <div class="contact__item">
       <i class="material-icons">&#xE88E;</i>
-      <div class="contact__info">{$contact_infos.details}</div>
+      <div class="contact__info">{$contact_infos.details|nl2br nofilter}</div>
```

Comment:
Instead of having this comment, you can do 
```
{if !empty($contact_infos.details)}
```

Reply:
@Hlavtox wdyt?